src/site/src/content/interventions.mdx

```diff
@@ -345,7 +345,138 @@ The adversarial debiasing technique is extremely effective for demographic parit
+
+</Collapse>
+
+<Collapse label="Experimental results">
+
+### Finance
+
+We applied the intervention to the adult dataset in order to impose different notions of fairness with respect to sex. Run our analysis yourself on Binder [here][binder-finance-agarwal]. 
+
+#### Demographic parity
+
+The intervention was very sucessful, reducing demographic from 0.193 to 0.013, while only sacrificing a small amount of accuracy, losing 1.5% from 85.3% to 83.8%.
+
+<PlotLoader source={"finance/interventions/agarwal-dp"} />
+
+#### Equalised odds
+
+The intervention again improved fairess massively, reducing equalised odds from 0.128 to 0.017, while almost no accuracy was lost, that is, accuracy only fell from 85.3% to 85.1%.
+
+<PlotLoader source={"finance/interventions/agarwal-eo"} />
+
+#### Equal opportunity
+
+Similarly concincing results were achieved for equal opportunity, reducing equalised odds from 0.128 to 0.014, while the accuracy actually rose by 0.2%. Note that in some cases the restricted state space for the model parameter due to the fairness constraints can lead to a minimiser converging at a local minimum preferable over the local minimum found on the unrestricted space.
+
+<PlotLoader source={"finance/interventions/agarwal-eopp"} />
+
+### Recruiting
+
+Here we applied the intervention to the synthetic recruiting dataset and impose different notions of fairness with respect to race.
+
+#### Demographic parity
+
+Due to the intervention demographic parity was significantly reduced from 0.327 to 0.041. However, the loss in accuracy was realtively high, reducing it from 86.2% to 80%.
+
+<PlotLoader source={"recruiting/interventions/agarwal-dp"} />
+
+#### Equalised odds
+
+Equalised odds could be roughly halved, from 0.133 to 0.065, which is a significant reduction. In this case, the loss in accuracy was small, with a final accuracy of 84.3% compared to 86.2%.
+
+<PlotLoader source={"recruiting/interventions/agarwal-eo"} />
+
+#### Equal opportunity
+
+The improvement in equal opportunity is similar to the improvement in equalised odds, reducing equal opportunity from 0.133 to 0.084, and similarly for accuracy, obtaining a fair model accuracy of 85.4%.
+
+<PlotLoader source={"recruiting/interventions/agarwal-eopp"} />
+
+</Collapse>
+
+### Summary
+
+The intervention improved fairness significantly on both data sets, while the overall observed trade-off between fairness and accuracy for the adult data set was superior compared to the synthetic recruiting data. Compared to the other intervention techniques Agarwal et al.'s intervention is highly competitive and leads to consistently good results in repeated independent experiments. For practical use this implementation hence seems to work well.
+
+
+
+
+Fairness through unawareness excludes protected attributes when learning a classifier with the aim on improving fairness. We consider the effect of applying FTU for a number of observational group fairness notions.
```

Comment:
Can we give this a title and also maybe move it to the top of the list of interventions? (Since it's the most basic)